ompi/mpi/man/man3/MPI_Barrier.md

```diff
@@ -0,0 +1,58 @@
+# NAME
+
+MPI_Barrier, MPI_Ibarrier - Synchronization between MPI processes in a group
+
+# Syntax
+
+## C Syntax
+```c
+    #include <mpi.h>
```

Comment:
You can probably remove the indenting on all 3 of the code blocks (i.e., just start them at the left margin).